src/main/resources/avro/common.avdl

```diff
@@ -2,6 +2,325 @@
 /**
 This protocol defines common types used in the other GA4GH protocols. It does
 not have any methods; it is merely a library of types.
+
```

Comment:
Also, a general comment about the documentation: words are fundamentally a bad way to describe graphical concepts. I'm glad to see the documentation come into place, but many of these concepts can be more clearly described via annotated illustrations.